### ISSUE TITLE
Report Types skew the Custom Fields utilization stats

### ISSUE THREAD

**Issue by vaumont** [SalesforceLabs/OrgCheck#60]
When an object is used in a Report Type, all its custom fields are flagged as being used by this RT; and, therefore, by all the reports that use this RT.
I find this misleading: RTs are not reliable indicators of whether nor not a custom field is used in a report.
I thing RT should be removed from the custom fields utilization chain.

**Comment by VinceFINET** [comment]
Is it the case only for custom report types and custom fields? or custom report types and any fields (custom or standard)?

**Comment by VinceFINET** [comment]
I added a custom field on Opportunity
I also created a custom report type for Opp and it included automatically this new field.

When I check the dependencies it looks like this:
![image](https://user-images.githubusercontent.com/3799275/112466690-683de700-8d66-11eb-954c-6890bde18cea.png)

I created two reports in my private folder:
- one with standard report type on Oppty and 
- one with that new custom report type on Oppty

After refreshed in **OrgCheck**, I have these dependencies for this field:
![image](https://user-images.githubusercontent.com/3799275/112467516-6e809300-8d67-11eb-97cc-79168adcd538.png)

Finally, if I remove the `Amount` custom field from the two previous reports, save them, refresh **OrgCheck** here is what I have in the dependencies in the tool for this field:
![image](https://user-images.githubusercontent.com/3799275/112467877-df27af80-8d67-11eb-9293-a83c50069277.png)

In conclusion, you require to have an option to exclude the "Custom Report Type" type when retrieving the dependencies for custom fields.
So that for example, a custom field that is used only in custom report type will have a bad score incremented.

**Comment by vaumont** [comment]
> a custom field that is used only in custom report type will have a bad score incremented.

How about discarding the ReportType dependency analysis all together?
Since that dependency includes all of the object's  custom fields, it's meaningless.
